commands/python/command/sync_dns.py

```diff
@@ -23,11 +24,19 @@ def sync_dns(namespace):
     if cr.is_local:
         logging.info("Not syncing DNS because you are on local")
         return
-    k = Kubernetes(namespace=namespace)
-    k_utils = KubernetesUtils(k)
-    nb_updated = cr.fill_hostedzone(
-        k_utils.get_services_to_load_balancers_map(
-            cr.dual_dns_prefix_annotation_name, cr.ingress_info
-        )
+
+    k_utils = KubernetesUtils(Kubernetes(namespace=namespace))
+
+    service_loadbalancers = k_utils.get_services_to_load_balancers_map(
+        cr.dual_dns_prefix_annotation_name, cr.ingress_info
     )
+
+    # Apply our dns to the service names
+    service_hostnames_to_loadbalancers = {
+        f"{service_name}.{cr.namespace_domain_name}": loadbalancer_hostname
+        for service_name, loadbalancer_hostname in service_loadbalancers.items()
+    }
```

Comment:
Brought this in from `ClusterRules.fill_hostedzone()` when it got moved to `dns_mapping`